src/modules/layout/components/Header.js

```diff
@@ -61,11 +61,35 @@ type PropsType = {|
   scenes: Array<NavigationScene>,
   t: TFunction,
   theme: ThemeType,
+  peeking: boolean | 'unsure',
```

Comment:
No idea what `unsure` means in this context. Does it mean `unknown'?

Reply:
It is like: true, false, unsure
or: ja, nein, jain :)

Reply:
If peeking is unsure then we the Header just has no information whether it is currently peeking. In that case it uses the cached value from the local state.

Reply:
Why don't you use `null` instead  of `unsure`?

Reply:
Maybe I could, yes but I like that way of expressing a trinary more.